cumulusci/robotframework/Salesforce.py

```diff
@@ -492,6 +493,38 @@ def salesforce_insert(self, obj_name, **kwargs):
         self.store_session_record(obj_name, res["id"])
         return res["id"]
 
+    def salesforce_collection_insert(self, objects):
+        def dict_to_insertable(d):
```

Comment:
There's no need for a closure here. So I'd rather this be a class method or module global to avoid defining it each time `salesforce_collection_insert` is called.

Reply:
I'd call that a premature optimization, considering that the code is more or less a bulk insert which takes (...checking...) 1/3 of a second not including any networking time. I'd prefer to keep the other namespaces uncluttered.

Reply:
Okay, you got me, I was using the performance argument against something that simply isn't to my taste. Given that it has full test coverage, I can live with it.